package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ValorToken",
-  "version": "1.0.0",
+  "version": "0.1.1",
```

Was it supposed to be version 1.0.1 ?
```suggestion
  "version": "1.0.1",
```

---

no we were on track for version 0.1.1 (a release candidate)